scripts/publish_github_docs.sh

```diff
@@ -49,7 +49,11 @@ fi
 find . \( -regex './[^/]*' -and -type f -and -not -path ./.git \) -or \( -path './search/*' \) -exec git rm -r {} +
 cp -ar "$DOX_DOCS/." .
 git add .
-gitcommit -m "Update docs to latest ($DESCRIBE)"
+if git diff --cached --quiet; then
+    echo "Docs already updated to the latest version, skipping..."
+else
+    gitcommit -m "Update docs to latest ($DESCRIBE)"
+fi
```

Comment:
Wow, so the implication is that between the 2020.2 and 2020.3 releases we didn't change anything that affected the doxygen? Weird.

Reply:
Ah no, the reality is weirder. Somehow the doc publishing step was successful once (moved to 2020.3) and as the script was re-run, doxygen generated the same output twice, hence no changes.